gpio_sim/setup_sims.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Creates GPIO-SIM entries that match the device-tree configuration
+# described in FILE
+
+# Get the directory of the sourced script
+script_dir=$(dirname "${BASH_SOURCE[0]}")
+
+# The device tree file to parse
+FILE=$script_dir/../device_trees/oresat-c3-0601.dts
```

Comment:
If this is running on an OreSat card, you can read from `/boot/dtbs/<kernel_verion>/am335x-pocketbeagle.dtbo` (it is symbolically linked to correct device tree) rather than hard-coding this